mapclassify/classifiers.py

```diff
@@ -1123,6 +1123,68 @@ def plot(
             plt.savefig(file_name, dpi=dpi)
         return f, ax
 
+    def plot_histogram(
+        self,
+        hist_color="dodgerblue",
+        linecolor="black",
+        ax=None,
+        despine=True,
+        hist_kwargs=None,
```

Comment:
Could this just be `**kwargs` so you don't have to define a dict when passing additional values?

Reply:
yup